source/rover.js

```diff
@@ -378,10 +397,13 @@ this.calcRam = function() {
 
 this.blockCountChanged = function( value ) {
     this.calcRam();
-    this.activateSensor( 'forward' );
+    this.activateSensor( 'anomaly' );
     this.activateSensor( 'signal' );
     this.activateSensor( 'heading', this.heading );
+    this.activateSensor( 'collision' );
+    this.activateSensor( 'proximity' );
```

Comment:
(looks like maybe you're already doing that elsewhere?  Ignore me if I'm confused :) )